.yarnrc

```diff
@@ -0,0 +1,2 @@
+# For .yarnrc
+registry=https://npm.pkg.github.com/yearn
```

Comment:
is this needed?

Reply:
It is for major lib when we implement

Reply:
Maybe we can add it then and not now, so it gets documented in same PR